### PR TITLE
use correct mean and standard deviation norm values in image tutorials

### LINKED ISSUE #50239: [Docs] Tutorials should use correct normalization values for image datasets
### Description

It is important for image datasets to use correct mean and standard deviation values ([more info here](https://www.richarddecal.com/data-science/use-these-normalization-values-for-torchvision-datasets/)). The Ray Docs often use inappropriate values for image datasets. For instance, [this tutorial](https://github.com/ray-project/ray/blob/27e58508950d5ff2a0cb042e46abc484ee4df912/doc/source/train/getting-started-pytorch-lightning.rst#L4) uses a mean and standard deviation of `0.5`, which are completely wrong for the FashionMNIST dataset.

### Link

_No response_

### PR DESCRIPTION
**Pull request by crypdick**
## Why are these changes needed?

Updates docs to use correct normalization values in image datasets.

## Related issue number

Closes #50239 

## Checks

- [ ] I've signed off every commit(by using the -s flag, i.e., `git commit -s`) in this PR.
- [ ] I've run `scripts/format.sh` to lint the changes in this PR.
- [ ] I've included any doc changes needed for https://docs.ray.io/en/master/.
    - [ ] I've added any new APIs to the API Reference. For example, if I added a 
           method in Tune, I've added it in `doc/source/tune/api/` under the 
           corresponding `.rst` file.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failures at https://flakey-tests.ray.io/
- Testing Strategy
   - [ ] Unit tests
   - [ ] Release tests
   - [ ] This PR is not tested :(

### CODE REVIEW

**Review by angelinalg — APPROVED** [review]
stamp

**Review by matthewdeng — APPROVED** [review]
